data_sources/bro_weird.yml

```diff
@@ -0,0 +1,16 @@
+name: Bro weird
+id: e03762c5-c4b8-44e3-b762-c5c4b8e4e3b6
+version: 2
+date: '2025-01-23'
+author: Jacob Delgado, SnapAttack
+description: Logs anomalous or unexpected network behaviors identified by Zeek (formerly
+  Bro), including protocol violations and unusual traffic patterns.
+mitre_components:
+- Network Traffic Flow
+- Network Traffic Content
+- Response Metadata
+- Application Log Content
+- Host Status
+source: bro:weird:json
```

Comment:
This sourcetype is not available in in the Zeek TA - https://splunkbase.splunk.com/app/5466